packages/fxa-admin-server/src/test/lib/db/models/accounts.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `accounts` (
```

Comment:
I don't suppose there's any way to sync this with the schema from `fxa-auth-db-mysql` is there? Probably not, since the schema there consists only of the 100+ patches to apply. Would be nice to test against the "real" thing though.

Reply:
Not that I could see, this schema doesn't actually exist in one place as-is, I had to extract the DDL statements to create the current table directly from MySQL.